### PR TITLE
Enable debug mode in helper script for integration tests

### PR DESCRIPTION
**Pull request by danxuliu**
Follow up to #12244

### CODE REVIEW

**Review by skalidindi53 — APPROVED** [review]

File: .github/workflows/integration-mariadb.yml [hunk 1]
```diff
@@ -147,7 +147,6 @@ jobs:
         run: |
           mkdir data
           ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
-          ./occ config:system:set debug --value=true --type=boolean
```
Comment (nickvergessen) [inline .github/workflows/integration-mariadb.yml:150]: lets keep it for others to copy

File: .github/workflows/integration-mariadb.yml [hunk 2]
```diff
@@ -147,7 +147,6 @@ jobs:
...
           ./occ config:system:set hashing_default_password --value=true --type=boolean
```
Comment (nickvergessen) [inline .github/workflows/integration-mariadb.yml:150]: that being said, we should copy this to the script as well, so locally uses the fast hashing as well

Comment (nickvergessen) [reply .github/workflows/integration-mariadb.yml:150]: and the federated server

File: .github/workflows/integration-mariadb.yml (hunk 1 above)
Comment (danxuliu) [reply .github/workflows/integration-mariadb.yml]: Commit dropped.

File: .github/workflows/integration-mariadb.yml (hunk 2 above)
Comment (danxuliu) [reply .github/workflows/integration-mariadb.yml]: Added in a new commit.

**Pull request merged**